ReactWindows/ReactNative/Storage/FutureAccessList.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.AccessCache;
+
+namespace ReactNative.Storage
+{
+    /// <summary>
+    /// The reason this future access list thing was introduced was a need
+    /// to transfer a file object from RN to JS and then to another RN module.
+    /// The naive approach to transfer the file path didn't work because the
+    /// file path doesn't carry read/write permissions and a UWP app can directly
+    /// access only files inside its app folder.
+    ///
+    /// Eventually, RNW will have a more general approach to store such things
+    /// and it will support other schemes like blob. At that time this class will
+    /// likely to be rewritten.
+    /// </summary>
+    public static class FutureAccessList
+    {
+        /// <summary>
+        /// FAL URIs start with this prefix.
+        /// </summary>
+        public static string Scheme => "urn:future-access-list:";
```

Comment:
>Scheme [](start = 29, length = 6)

Placeholder comment, does this need to be public?

Reply:
Making this field `private` would be a false impression that this string can be changed or that nobody depends on it. In fact, a practical implementation of FAL needs changes in JS to recognize such URIs, e.g. if a URI starts with this prefix it's sent to FAL, otherwise somewhere else.

Reply:
That's a good point.